src/components/2019/year-in-review/Typograph.js

```diff
@@ -0,0 +1,126 @@
+import s from 'styled-components'
+import { OPEN_SANS_REGULAR, OPEN_SANS_BOLD, DM_SERIF_DISPLAY_BOLD, DM_SERIF_DISPLAY_REGULAR } from '../../../utils/font'
+
+export const SectionHeader = s.h1`
+    color: ${({ color }) => color};
+    ${DM_SERIF_DISPLAY_BOLD}
+    font-size: 52px;
+    font-style: italic;
+    text-align: center;
+    margin-bottom: 40px;
+`
+
+export const SectionDescription = s.p`
+    ${OPEN_SANS_REGULAR}
+    max-width: 60%;
+    margin: 16px auto 64px;
+    text-align: center;
+`
+
+export const MoreStories = s.div`
+    margin-top: 20px;
+    padding-top: 20px;
+    background-color: inherit;
+    border-top: 1px solid ${({ inverse }) => inverse ? "rgba(0, 0, 0, 0.3)" : "rgba(255, 255, 255, 0.3)"}; 
+`
+
+export const PrimaryStoryLine = s.hr`
+    background-color: ${({ inverse }) => inverse ? "rgba(0, 0, 0, 0.3)" : "rgba(255, 255, 255, 0.3)"};
+    margin: 28px 0px;
+`
+
+export const StyledStoryLink = s.a`
+    color: inherit;
+
+    :hover {
+        text-decoration: none;
+        color: inherit;
+    }
+`
+
+export const PrimaryStory = s.h4`
+    color: ${({ inverse }) => inverse ? "#131313" : "white"};
+    ${DM_SERIF_DISPLAY_REGULAR}
+    font-size: 22px;
+
+    :hover {
+        color: ${({ inverse }) => inverse ? "#4a4a4a" : "#dbdbdb"};
+    }
+`
+
+export const SecondaryStory = s.h5`
+    color: ${({ inverse }) => inverse ? "#131313" : "white"};
```

Comment:
this line is used in so many places in the code that I think it violates DRY, can you think up a solution to not repeat it?